### PR TITLE
Add Cameron's suggestions from PR #14

### PR DESCRIPTION
**Pull request by jenna-tomkinson**
This PR is due to a commit mistake on my end. It should be fixed now, thanks to @d33bs!

### CODE REVIEW

File: 2.cellprofiler_analysis/nf1_analysis.ipynb [hunk 1]
```diff
@@ -36,9 +36,7 @@
    "source": [
     "## Set paths for each plate\n",
     "\n",
-    "Note: Due to the difference channel numbers between plates 1 + 2 (3 channels) and plates 3 + 3 prime (4 channels), there needs to be two difference cppipe files (like in the IC module). \n",
-    "\n",
-    "As well, the output file path does not need to be strict since the `run_cellprofiler` function can create the output folder directory if it doesn't already exist. The other paths must be strict since these files should already exist for CellProfiler to run. The output directory doesn't need to already exist."
+    "Note: Due to the difference channel numbers between plates 1 + 2 (3 channels) and plates 3 + 3 prime (4 channels), there needs to be two different cppipe files (like in the IC module). "
```
Comment (d33bs) [inline 2.cellprofiler_analysis/nf1_analysis.ipynb]:
```suggestion
    "Note: Due to the different channel numbers between plates 1 + 2 (3 channels) and plates 3 + 3 prime (4 channels), there needs to be two different cppipe files (like in the IC module). "
```

**Review by d33bs — APPROVED** [review]
Nice job! Left a minor comment, otherwise LGTM!

File: 2.cellprofiler_analysis/nf1_analysis.ipynb (hunk 1 above)
Comment (jenna-tomkinson) [reply 2.cellprofiler_analysis/nf1_analysis.ipynb]: Fixed!

**Comment by jenna-tomkinson** [comment]
@d33bs Thank you for the help! All commits are made and I will be merging 😄

**Pull request merged**